android/src/main/java/org/reactnative/MutableImage.java

```diff
@@ -1,282 +0,0 @@
-package org.reactnative;
```

Comment:
is this used on RCTCamera?

Reply:
No. RCTCamera has its own MutableImage file in its own package.